core/src/repair/ancestor_hashes_service.rs

```diff
@@ -1213,9 +1204,14 @@ mod test {
         assert!(dead_slot_pool.contains(&dead_slot));
         assert!(repairable_dead_slot_pool.is_empty());
 
+        let validator_stakes: ValidatorStakesMap = (0..2)
+            .zip(vote_simulator.node_pubkeys.iter())
+            .map(|(_i, pk)| (*pk, 42))
+            .collect();
+        cluster_slots.fake_epoch_info_for_tests(validator_stakes);
```

Comment:
Due to the change in ClusterSlots we now need to insert some epoch info into the datastructure before we can expect it to do meaningful things. This code achieves the necessary patching.